### DISCUSSION TITLE
[Feature] Person Sharing

### DISCUSSION THREAD

**Discussion by emabiz** [immich-app/immich#6755] [Feature Request]
### The feature

Hi,
I'd like to share not only albums, but also people, so that all photos containing one person are visible to other users.
Do you think it is possible to have?

Thank you

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by jjspierx** [comment]
This would be a nice feature for sure.  My wife and I used this back in the old days of that other app, so I didn't have to see every picture she took in my timeline, but all the pictures of ourselves and our kids were automatically shared between us.

**Comment by Sums123** [comment]
This is being worked on
https://github.com/immich-app/immich/discussions/1673#discussioncomment-8326841

**Reply by martabal** [reply]
No, this is a different feature

**Reply by bo0tzz** [reply]
Album rules will make this feature possible (just create a rule and then share the resulting album).

**Reply by martabal** [reply]
Of course, but non-owners won't be able to see people in the detail-panel

**Comment by bo0tzz** [comment]
#12614
